windows-agent/internal/config/config_test.go

```diff
@@ -411,9 +432,17 @@ func TestSetUserLandscapeConfig(t *testing.T) {
 
 			got, src, err := conf.LandscapeClientConfig()
 			require.NoError(t, err, "LandscapeClientConfig should return no errors")
-			require.Equal(t, landscapeConfig, got, "Did not get the same value for landscape config as we set")
-			require.Equal(t, config.SourceUser, src, "Did not get the same value for landscape config as we set")
+			require.Equal(t, wantSource, src, "Did not get the same source for landscape config as we set")
 			require.Equal(t, 1, calledLandscapeNotifier, "LandscapeNotifier should have been called once")
+
+			want := landscapeBaseConf
+			if tc.settingsState.is(landscapeUIDHasValue) {
+				want += "\nhostagent_uid=landscapeUID1234\n"
+			}
+			if wantSource == config.SourceNone {
```

Comment:
I think if source is `SourceNone`, it means that we should discare the "got" value, no? Shouldn’t then just return early to end testing, even before this all `want` dance? (doing it on line 437)

Reply:
Good point.